wallet/src/Pos/Util/Mnemonic.hs

```diff
@@ -294,15 +295,22 @@ instance Buildable (SecureLog (Mnemonic mw)) where
     build _ =
         "<mnemonic>"
 
-instance Buildable MnemonicErr where
+instance Buildable (MnemonicError csz) where
     build = \case
-        MnemonicErrInvalidEntropyLength l ->
-            bprint ("Entropy must be a sequence of " % build % " bytes") l
-        MnemonicErrFailedToCreate ->
-            bprint "Invalid Mnemonic words"
-        MnemonicErrForbiddenMnemonic ->
+        ErrMnemonicWords (ErrWrongNumberOfWords a e) ->
+            bprint ("Invalid number of mnemonic words: got "%build%" words, expected "%build%" words") a e
+        ErrDictionary (ErrInvalidDictionaryWord w) ->
+            bprint ("Invalid dictionary word: "%build%"") (fromUtf8String w)
+        ErrEntropy (ErrInvalidEntropyLength a e) ->
+            bprint ("Invalid entropy length: got "%build%" bits, expected "%build%" bits") a e
+        ErrEntropy (ErrInvalidEntropyChecksum a e) ->
+            bprint ("Invalid entropy checksum: got "%build%", expected "%build) (show' a) (show' e)
```

Comment:
Some day ago I thought about whether one should provide any more explanation or suggestions here. Like mention "mnemonic" at least. I wouldn't be confident to do it now though, and no time now.